static/app/views/settings/organizationDeveloperSettings/resourceSubscriptions.tsx

```diff
@@ -27,7 +26,7 @@ export default class Subscriptions extends Component<Props> {
     webhookDisabled: false,
   };
 
-  constructor(props: Props, context: Context) {
+  constructor(props: Props, context) {
```

Comment:
is there any other type we can use here for context?

Reply:
I'm not sure tbh

Reply:
This was just wrong though